install/bin/decj

```diff
@@ -50,43 +50,9 @@ fi
 # Build classpath
 CP=$DH/bin/opendcs.jar:
 
-# # For CWMS Installations, add the HEC and RMA Jars.
-# if [ -n "$CWMS_JAR_DIR" -a -d "$CWMS_JAR_DIR" ]
-# then
-#   CP=$CP:$CWMS_JAR_DIR/hec.jar:$CWMS_JAR_DIR/hecData.jar:$CWMS_JAR_DIR/heclib.jar:$CWMS_JAR_DIR/rma.jar:$CWMS_JAR_DIR/lookup.jar
-# fi
-# if [ -n "$CWMS_SYSJAR_DIR" -a -d "$CWMS_SYSJAR_DIR" ]
-# then
-#   CP=$CP:$CWMS_SYSJAR_DIR/jdom.jar:$CWMS_SYSJAR_DIR/jdom-1.0.jar
-# fi
-
-
-if [ ! -z "$CP_SHARED_JAR_DIR" -a -d "$CP_SHARED_JAR_DIR" ]
```

Comment:
@agilmore2 where these for HDB, the ones above that says CWMS are rather obvious and we don't need them anymore but I'm not sure if I went to far with these.